cms/static/js/views/active_video_upload_list.js

```diff
@@ -10,6 +11,7 @@ define(
                 "dragleave .file-drop-area": "dragleave",
                 "drop .file-drop-area": "dragleave"
             },
+            pageUnloadMessage: pageUnloadMessage,
```

Comment:
No, I guess my comment was unclear. The test you have is fine. I'm suggesting that you do the following:
```js
var ActiveVideoUploadListView = BaseView.extend({
    pageUnloadMessage: gettext("Your video uploads are not complete."),
    tagName: "div",
    // etc...
    onBeforeUnload: function () {
        // body of function here...
        if (uploading.length) {
            return this.pageUnloadMessage;
        }
    }
})```
Does that make sense?